### DISCUSSION TITLE
Upload video as a sprite

### DISCUSSION THREAD

**Discussion by WoolyMammothMC** [ScratchAddons/ScratchAddons#7275] [Ideas]
Just like how you can upload a .gif as a sprite and it will automatically put the frames from the .gif as costumes in order from the start to end, I think it would be great to have an addon in where you can upload an .mp4 as a sprite and it will put all the frames from the video as costumes in order from start to finish, with the audio obviously being uploaded to the sprite as a sound, with maybe some inbuilt code "when flag clicked: forever{next costume}" and maybe in the extension settings you can choose what is uploaded when u upload the .mp4, you can whether you want the in built code or not and whether you want the audio or not.

**Comment by GoatTech-42** [comment]
Would there be a FPS measurement, or just 30?

**Comment by Samq64** [comment]
Duplicate of #7276
